docs/advanced_topics/accessibility_considerations.md

```diff
@@ -126,6 +126,18 @@ Make sure to test your forms’ implementation with assistive technologies, and
 
 Here are things you can do to help authors create accessible content.
 
+### built-in accessibility checker
```

Comment:
The built-in accessibility checker isn't really a "thing you can do", so I'd suggest either re-wording the sentence above, or moving this to its own second-level section.

Also, "Built-in" should be capitalised:
```suggestion
### Built-in accessibility checker
```

Reply:
Like "The process of creating accessible content can be enhanced with the use of built-in tools and additional resources."?

Reply:
Sounds good! But maybe a bit less wordy - "A number of built-in tools and additional resources are available to help create accessible content."